catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -74,6 +75,18 @@ const WorkDetails = ({
   itemsRef.current = itemsWithPhysicalLocations;
   const [hasRequestableItems, setHasRequestableItems] = useState(false);
   const singleItem = itemsWithPhysicalLocations.length === 1;
+  const [imageJSON, setImageJSON] = useState(null);
+  const fetchImageJSON = async () => {
```

Comment:
I'd prefer `fetchImageJson`, `imageJson` and `setImageJson` (i.e. not all caps for `JSON`), but I appreciate that that might be getting a bit bike-sheddy!